### PR TITLE
New Heat Exchanger

### PR DESCRIPTION
**Pull request by frmanso**
New water pipes, grooves and manifolds.

### CODE REVIEW

**Comment by matthiasrichter** [comment]
The `modernize-use-default-member-init` errors of the code checker suggest to specify default values in the member declaration in the header file.

There is no decision yet how to deal with that, it started a while ago with an update of `clang-format`. Should be discussed in WP3 with @dberzano 

I'm puzzled by the fact that the diff for `HeatExchanger.cxx` shows the complete file being changed, that makes it difficult to inspect the differences. Did you do anything special with this file?

**Comment by frmanso** [comment]
Hello,
    I made a complete review of the volumes used to build the heat exchanger. There are new water pipes, addition of grooves inside the rohacell plates. Another people worked on manifolds with a lot of new details and has added severals new layers (glues and kapton). This is really a new geometry that has been review in WP9 for the MFT collaboration. With Anotnio Uras, I'm one of the convener of WP9 in charge of the MFTgeometry.

File: Detectors/ITSMFT/MFT/base/include/MFTBase/Geometry.h [hunk 1]
```diff
@@ -76,7 +76,9 @@ class Geometry : public TNamed
   static const Double_t sGlueEdge;
   static const Double_t sShiftDDGNDline;
   static const Double_t sShiftline;
-  static const Double_t sRohacell;
+  static const Double_t mGlueRohacellCarbonThickness;
+  static const Double_t mKaptonOnCarbonThickness;
+  static const Double_t mKaptonGlueThickness;
```
Comment (sawenzel) [inline Detectors/ITSMFT/MFT/base/include/MFTBase/Geometry.h]: please use `s` and not `m` for these variables.

File: Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx [hunk 2]
```diff
@@ -1,2829 +1,3872 @@
...
-    new TGeoCombiTrans(0., 0., mZPlan[disk] - deltaz / 2. + mCarbonThickness + mRWater + mDRPipe, rotation);
-  mHalfDisk->AddNode(cooling, 4, transformation);
-  //  }
-
-  // **************************************** Carbon Plates ****************************************
-
-  auto* carbonPlate = new TGeoVolumeAssembly(Form("carbonPlate_D2_H%d", half));
-
-  auto* carbonBase2 = new TGeoBBox(Form("carbonBase2_D2_H%d", half), (mSupportXDimensions[disk][0]) / 2.,
-                                   (mSupportYDimensions[disk][0
```
Comment (matthiasrichter) [inline Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx]: the encoding of the file was changed to `dos`, that's why it's completely different.

File: Detectors/ITSMFT/MFT/base/include/MFTBase/Geometry.h (hunk 1 above)
Comment (frmanso) [reply Detectors/ITSMFT/MFT/base/include/MFTBase/Geometry.h]: Hello, it's done now.

File: Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx (hunk 2 above)
Comment (frmanso) [reply Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx]: What should I do? I'm working on files using a same emacs session. Is that an issue coming from my github interface instead?

Comment (frmanso) [reply Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx]: Should be fine now...

Comment (sawenzel) [reply Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx]: Thanks

**Pull request merged**